InAppMessaging/Example/Tests/FIRIAMDisplayExecutorTests.m

```diff
@@ -79,35 +92,44 @@ typedef NS_ENUM(NSInteger, FIRInAppMessagingDelegateInteraction) {
   FIRInAppMessagingDelegateInteractionError,  // message display component triggers
   // displayErrorEncountered:
   FIRInAppMessagingDelegateInteractionImpressionDetected,  // message has finished a valid
-                                                           // impression, but it's not getting
-                                                           // closed by the user.
+  // impression, but it's not getting
+  // closed by the user.
 };
 
 // A class implementing protocol FIRInAppMessagingDisplay to be used for unit testing
 @interface FIRIAMMessageDisplayForTesting : NSObject <FIRInAppMessagingDisplay>
 @property FIRInAppMessagingDelegateInteraction delegateInteraction;
+@property(nonatomic, nullable, copy) FIRInAppMessagingAction *action;
 
 // used for interaction verificatio
 @property FIRInAppMessagingDisplayMessage *message;
+- (instancetype)initWithDelegateInteraction:(FIRInAppMessagingDelegateInteraction)interaction
+                                     action:(nullable FIRInAppMessagingAction *)actionURL;
 - (instancetype)initWithDelegateInteraction:(FIRInAppMessagingDelegateInteraction)interaction;
 @end
 
 @implementation FIRIAMMessageDisplayForTesting
-- (instancetype)initWithDelegateInteraction:(FIRInAppMessagingDelegateInteraction)interaction {
+- (instancetype)initWithDelegateInteraction:(FIRInAppMessagingDelegateInteraction)interaction
+                                     action:(nullable FIRInAppMessagingAction *)action {
   if (self = [super init]) {
     _delegateInteraction = interaction;
+    _action = action;
   }
   return self;
 }
 
+- (instancetype)initWithDelegateInteraction:(FIRInAppMessagingDelegateInteraction)interaction {
+  return [self initWithDelegateInteraction:interaction action:nil];
+}
+
 - (void)displayMessage:(FIRInAppMessagingDisplayMessage *)messageForDisplay
        displayDelegate:(id<FIRInAppMessagingDisplayDelegate>)displayDelegate {
   self.message = messageForDisplay;
 
   switch (self.delegateInteraction) {
-    case FIRInAppMessagingDelegateInteractionClick:
-      [displayDelegate messageClicked:messageForDisplay];
-      break;
+    case FIRInAppMessagingDelegateInteractionClick: {
```

Comment:
What's the purpose of the `{` and `}` here?

Reply:
Removed.